src/elements/content-sidebar/activity-feed/activity-feed/ActivityThreadReplyForm.js

```diff
@@ -0,0 +1,57 @@
+// @flow
+
+import * as React from 'react';
+import { FormattedMessage, useIntl } from 'react-intl';
+
+import PlainButton from '../../../../components/plain-button';
+import ArrowArcRight from '../../../../icon/fill/ArrowArcRight';
+
+import type { GetProfileUrlCallback } from '../../../common/flowTypes';
+import type { SelectorItems, User } from '../../../../common/types/core';
+
+import CommentForm from '../comment-form';
+
+import messages from './messages';
+import './ActivityThreadReplyForm.scss';
+
+type Props = {
+    currentUser?: User,
+    getMentionWithQuery?: Function,
+    getUserProfileUrl?: GetProfileUrlCallback,
+    mentionSelectorContacts?: SelectorItems<>,
+    onReplyCreate: Function,
+};
+
+export default function ActivityThreadReplyForm({
+    currentUser,
+    mentionSelectorContacts,
+    getMentionWithQuery,
+    onReplyCreate,
+}: Props) {
+    const [showReplyForm, setShowReplyForm] = React.useState(false);
+    const placeholder = useIntl().formatMessage(messages.replyInThread);
+
+    return showReplyForm ? (
+        <CommentForm
+            className=""
+            isOpen
+            isEditing
+            getAvatarUrl={() => Promise.resolve()}
```

Comment:
What is the goal of this change?

Reply:
As per design I don't need to show an avatar (thus `isEditing`), but in CommentForm props `getAvatarUrl` is defined as required 
https://github.com/box/box-ui-elements/blob/6b59624dcd33f95eb24193fce1683948f8fcf6c1/src/elements/content-sidebar/activity-feed/comment-form/CommentForm.js#L30
where `GetAvatarUrlCallback` is defined as
https://github.com/box/box-ui-elements/blob/6b59624dcd33f95eb24193fce1683948f8fcf6c1/src/elements/common/flowTypes.js#L25
I wanted to avoid unnecessary props for `AvtivityThreadReplyForm` and prop drilling form above.